src/lib.rs

```diff
@@ -106,6 +113,26 @@ impl Encoder<ClamdRequestMessage> for ClamdZeroDelimitedCodec {
                 dst.put_u32(0);
                 Ok(())
             }
+            ClamdRequestMessage::StartSession => {
+                dst.reserve(10);
+                dst.put(&b"zIDSESSION"[..]);
+                dst.put_u8(0);
+                Ok(())
+            }
+            ClamdRequestMessage::EndSession => {
+                dst.reserve(10);
+                dst.put(&b"zEND"[..]);
+                dst.put_u8(0);
+                Ok(())
+            }
+            ClamdRequestMessage::ContScan(path) => {
+                dst.reserve(10);
+                dst.put(&b"zCONTSCAN "[..]);
+                // TODO: safety
```

Comment:
I guess there is also no method for `ContScan` on the `ClamdClient` so we can just keep it this way for now